Start/readme2.md

```diff
@@ -0,0 +1,55 @@
+## 4.3.3 &ensp; Users Perspective: The Front View
+The various aspects an users expects from the system and which is my one of the main ideal outcomes of the project can be viewed from the following figure:
+
+<p align=center>
+<img src="Start\user perspective.png" height=390px width=390px>
+</p>
+
+<br>
+&ensp; The above figure portrays the 
+exibilities and accessibilities provided to the users in order to
+perform various operations on the properties included in the databases. These are:
+
+<br>
+<br>
+<b>&ensp;&ensp;
+1. <i>View:</i>
```

Comment:
@smriti1313 ```<i>View:</i> ``` This shows that you have to explore more on markdown formats.

Reply:
Hi, Sorry. I will solve all this by tomorrow.